providers-sdk/v1/upstream/mvd/cvss/cvss.go

```diff
@@ -118,6 +121,91 @@ func init() {
 			"M", "L", "M", "H", "ND",
 		},
 	}
+	// MetricsV4 defines the valid CVSS 4.0 metrics and their allowed values.
+	// CVSS v4 https://www.first.org/cvss/v4/specification-document
```

Comment:
This link didn't work for me.
I had to use this: https://www.first.org/cvss/v4-0/specification-document